### PR TITLE
Implement Magic

### PR DESCRIPTION
**Pull request by S-Mackenzie1678**

### CODE REVIEW

File: entities/FireAleph.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	void use(PlayerCharacter player) {
+		// Have say not usable
+	}
+	
+	void useInBattle(PlayerCharacter player, Opponent enemy) {
+		if(mpCheck(player)) {			
+	  	  if(enemy.type == "grass" || enemy.type == "ice") {
+		  	enemy.hp -= 3;
+		  } else if(enemy.type == "water") {
+		  	enemy.hp -= 0;
```
Comment (adamhutchings) [inline entities/FireAleph.java]: Redundant line. Maybe say, after the initial if, else if (enemy.type != water) {enemy.hp -= 1} and then nothing else?

File: entities/FireAleph.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	int itemID = -1001;
+	
+	int mpCost = 1;
+	
+	void use(PlayerCharacter player) {
+		// Have say not usable
+	}
+	
+	void useInBattle(PlayerCharacter player, Opponent enemy) {
+		if(mpCheck(player)) {
```
Comment (adamhutchings) [inline entities/FireAleph.java:16]: mpCheck() is not defined.

Comment (adamhutchings) [reply entities/FireAleph.java:16]: Sorry, never mind.

File: entities/PlayerCharacter.java [hunk 3]
```diff
@@ -8,17 +8,21 @@
...
+    int xp;
     int strength;
     int defense;
     int magic;
     int resistance;
     int fleetness;
     int accuracy;
 
-    public PlayerCharacter(String name, boolean gender, int hp, int strength, int defense, int magic, int resistance, int fleetness, int accuracy) {
+    public PlayerCharacter(String name, boolean gender, int hp, int mp, int xp, int strength, int defense, int magic, int resistance, int fleetness, int accuracy) {
```
Comment (adamhutchings) [inline entities/PlayerCharacter.java:20]: These constructors be getting a little too long. Maybe set all the stats outside of the constructor? But not sure.

File: entities/FireAleph.java (hunk 1 above)
Comment (S-Mackenzie1678) [reply entities/FireAleph.java]: Oh that's big facts

**Review by adamhutchings — APPROVED** [review]
Looks good!

**Pull request merged**